source/index.html.md

```diff
@@ -132,14 +132,257 @@ curl "https://www.castupload.com/api/v1/actor_profiles/123" \
+  },
+  "drivers_licenses": [
+    "pkw-klasse-b-be"
+  ],
+  "licenses": [
+    "fallschirmspringer-lizenz"
+  ],
+  "country": "DE",
+  "state": "DE-RP",
+  "pictures": [
+    {
+      "id": 140,
+      "copyright": "Jane Doe",
+      "created_at": "2021-06-21T16:18:41.087+02:00",
+      "url": "http://static.castupload.com/development/b6ac24ef-7bbc-4486-bfc9-89fdce3a9fde.png",
+      "versions": {
+        "original": "http://static.castupload.com/development/b6ac24ef-7bbc-4486-bfc9-89fdce3a9fde.png",
+        "large": "https://d205pfv3qf1itp.cloudfront.net/s3/castupload-com/development/b6ac24ef-7bbc-4486-bfc9-89fdce3a9fde.png/755095922e94f912e6691c63f7aef9887d069f13c5dc84b581bbabea6761f7d5/c_limit,f_jpg,h_1200,q_80,w_1200",
+        "thumb": "https://d205pfv3qf1itp.cloudfront.net/s3/castupload-com/development/b6ac24ef-7bbc-4486-bfc9-89fdce3a9fde.png/8cbb163eb48e440b5e9715e0c551180817f76c7072fd205df3c320bae08548e8/c_crop,f_jpg,h_370,q_80,w_370,x_0,y_0/c_fill,g_North,h_190,w_190"
+      },
+      "orientation": "square",
+      "dimensions": [
+        370,
+        370
+      ],
+      "crop_coordinates": {
+        "x": 0,
+        "y": 0,
+        "w": 500,
+        "h": 500
+      },
+      "main_picture": true,
+      "year": 2021,
+      "recorded_at": "2020-06-22T11:51:51.363+02:00"
+    },
+    {
+      "id": 141,
+      "copyright": "",
+      "created_at": "2021-06-21T16:18:47.460+02:00",
+      "url": "http://static.castupload.com/development/5c5a1832-3ee4-495d-bfb0-03694dab0bd2.png",
+      "versions": {
+        "original": "http://static.castupload.com/development/5c5a1832-3ee4-495d-bfb0-03694dab0bd2.png",
+        "large": "https://d205pfv3qf1itp.cloudfront.net/s3/castupload-com/development/5c5a1832-3ee4-495d-bfb0-03694dab0bd2.png/c323525480481e940c6b95eced100afa1b3c23ef2af4431df592b0e492de2211/c_limit,f_jpg,h_1200,q_80,w_1200",
+        "thumb": "https://d205pfv3qf1itp.cloudfront.net/s3/castupload-com/development/5c5a1832-3ee4-495d-bfb0-03694dab0bd2.png/e63279986f8a8b40f0d1a99bb14c69918a749a34750248578a95ee17fdc58da7/c_fill,f_jpg,g_North,h_190,q_80,w_190"
+      },
+      "orientation": "landscape",
+      "dimensions": [
+        365,
+        360
+      ],
+      "main_picture": false,
+      "year": 2021,
+      "recorded_at": null
+    }
+  ],
+  "external_showreel": "https://www.youtube.com/watch?v=xxxx",
+  "external_showreels": [
+    "https://www.youtube.com/watch?v=xxxx",
+    "https://vimeo.com/xxxx"
+  ],
+  "showreels": [
+    {
+      "name": "Showreel 2021",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/15",
+      "type": "video"
+    },
+    {
+      "name": "Showreel 2020",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/16",
+      "type": "video"
+    }
+  ],
+  "vita": {
+    "primary_education": [
+      {
+        "name": "ABC School",
+        "year_from": 2012,
+        "id": "Z2lkOi8vY2FzdHVwbG9hZC9BY3RpbmdTY2hvb2w6OkNvbm5lY3Rpb24vMg"
```

Comment:
what id is this?

Reply:
This came from subject "casting_system". I removed all fields where this is checked.

Reply:
As I wrote before - we should only include information from the "public" facing API (i.e. no admin rights, no casting system subjects).